src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -335,8 +392,9 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
           Delaunay.Vector2(rotatedVertex(0), rotatedVertex(1))
         }.distinct.toList
 
-        // If z-component of normal is negative, order needs to be reversed to maintain clockwise orientation
-        if (face.c < -NumericUtils.EPSILON) {
+        // If normal is -z-axis, order needs to be reversed to maintain clockwise orientation
+        // rotation matrix is identity matrix in this case
+        if (math.abs(face.c + 1.0) < NumericUtils.EPSILON) {
```

Comment:
Great! Do you think you could add a short comment about that, just for those of us who aren't as geometrically inclined? :)